### ISSUE TITLE
Project status?

### ISSUE THREAD

**Issue by MrHaroldA** [codewombat/backshop#1]
I'm currently trying to configure a Drupal Commerce Kickstart site, but I find is very difficult up to "absurdly difficult" just to create some shipping cost rules.

So I'm now looking into easier alternatives, which brought me here.

What's the state of backshop right now; is it usable for a simple webshop that needs:
- Cart
- Checkout process
- Shipping costs; based on location, weight and size.
- Paypal payment
- Bank transfer (manual check)

And this would be nice to have to:
- Stock
- Discount; either through a predefined code, or based on which payment method the customer chose

I have a limited amount of time available, so I can't promise to help you with all missing, or incomplete features.

**Comment by MrHaroldA** [comment]
Ok, I checked out the latest revision and installed in Backdrop. looks like there's some stuff left to do ;)

I'll try to look into some issues; but I guess this is too much work for me to tackle.

**Comment by thamas** [comment]
@rszrama @MrHaroldA FYI https://github.com/backdrop-ops/contrib/issues/34#issuecomment-216664916

**Comment by laryn** [comment]
Update from a short Twitter exchange with @rszrama a few months back:

> Sadly, I haven't had time for it since taking over Commerce Guys, but Backdrop itself is maturing in useful directions on its own.
> Stalled in early stages; there was no Address Field or Entity Reference then; the former exists now, latter is still a core W.I.P.

@jenlampton wrote on Gitter:

>  Ryan was waiting on a reference module, and we now have https://github.com/backdrop-contrib/reference
> when that matures it will be moved into core, but there's no reason backshop can't start using it today
